app/src/main/res/layout/search_widget_small_no_mic.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!-- FrameLayout is needed to properly align the widget in the home screen. -->
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <FrameLayout android:layout_width="100dp"
+            android:layout_height="50dp"
+            android:background="@drawable/rounded_white_corners"
+            android:layout_gravity="center"
+            android:orientation="vertical">
+
+        <ImageButton
+                android:id="@id/button_search_widget_new_tab"
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:background="@drawable/ic_logo_widget"
+                android:layout_gravity="center"
+                android:layout_marginTop="5dp"/>
+    </FrameLayout>
+
+</FrameLayout>
```

Comment:
You'll need a new line at the end of this file or the linter will complain I believe.